adding in faq (#153)
stackblitz/docs: docs/guides/user-guide/general-faqs.md

# General FAQs

### Why do I have to Re-Authorize to access GitHub repositories?

> As of September 27th, 2023 we've begun a gradual migration from GitHub's OAuth app to their GitHub App. We received a lot of feedback from users saying that they didn't like the *all* or nothing approach that GitHub's OAuth app takes so we've started switching over to the GitHub app itself which allows per organization granularity.

Reviewer: more like per repository granularity, because with Oauth you can do installation into an org, but no to a specific repo

> As of September 27th, 2023 we've begun a gradual migration from GitHub's OAuth app to their GitHub App. We received a lot of feedback from users saying that they didn't like the *all* or nothing approach that GitHub's OAuth app takes so we've started switching over to the GitHub app itself which allows per organization granularity.

Reviewer 2: ```suggestion
As of September 27th, 2023 we started migrating from GitHub's OAuth app to their GitHub App. We received a lot of feedback from users saying that they didn't like the "*all* or nothing" approach that GitHub's OAuth app takes so we've started switching over to the GitHub app itself which allows granularity on a per-organization basis.
```

> As of September 27th, 2023 we've begun a gradual migration from GitHub's OAuth app to their GitHub App. We received a lot of feedback from users saying that they didn't like the *all* or nothing approach that GitHub's OAuth app takes so we've started switching over to the GitHub app itself which allows per organization granularity.

Author: ```suggestion
As of September 27th, 2023 we've begun a gradual migration from GitHub's OAuth app to their GitHub App. We received a lot of feedback from users saying that they didn't like the *all* or nothing approach that GitHub's OAuth app takes so we've started switching over to the GitHub app itself which allows per repository granularity.
```

> As of September 27th, 2023 we started migrating from GitHub's OAuth app to their GitHub App. We received a lot of feedback from users saying that they didn't like the "*all* or nothing" approach that GitHub's OAuth app takes so we've started switching over to the GitHub app itself which allows granularity on a per-organization basis.

Author: ```suggestion
As of September 27th, 2023 we started migrating from GitHub's OAuth app to their GitHub App. We received a lot of feedback from users saying that they didn't like the "*all* or nothing" approach that GitHub's OAuth app takes so we've started switching over to the GitHub app itself which allows granularity on a per-repository basis.
```

> In order to make this migration as smooth as possible we converted our "Codeflow bot" app to just a "StackBlitz" app. We also had to adjust the permissions to enable us to:

Reviewer 2: ```suggestion
To ensure a smooth migration, we converted our "Codeflow bot" app into a "StackBlitz" app. Additionally, we made necessary adjustments to the permissions to enable us to:
```

- Create repositories and manage
- Import teams to StackBlitz
- Login to StackBlitz

> if you have any questions about this proccess feel free to reach out to [support@stackblitz.com](mailto:support@stackblitz.com)

Reviewer 2: ```suggestion
If you have any questions about this process feel free to reach out to [support@stackblitz.com](mailto:support@stackblitz.com)
```
